mappings/net/minecraft/world/EntityView.mapping

```diff
@@ -76,19 +76,39 @@ CLASS net/minecraft/class_1924 net/minecraft/world/EntityView
 		ARG 1 entityClass
 		ARG 2 box
 		ARG 3 predicate
-	METHOD method_8333 getEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+	METHOD method_8333 getSurroundingEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;Ljava/util/function/Predicate;)Ljava/util/List;
+		COMMENT Computes a list of entities surrounding the given entity within some box.
+		COMMENT
+		COMMENT @return a list of entities within a box, excluding the given entity, all satisfying the given predicate
 		ARG 1 except
+			COMMENT the entity the box logically surrounds. This entity is ignored if it is inside the box.
 		ARG 2 box
+			COMMENT the box in which to search for entities
 		ARG 3 predicate
-	METHOD method_8335 getEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;)Ljava/util/List;
+			COMMENT a predicate which entities must satisfy in order to be included in the returned list.
+	METHOD method_8335 getSurroundingEntities (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;)Ljava/util/List;
```

Comment:
```suggestion
	METHOD method_8335 getEntitiesExcept (Lnet/minecraft/class_1297;Lnet/minecraft/class_238;)Ljava/util/List;
```

Reply:
More like `getEntitiesAround`

Reply:
Hmm, I am thinking about `getOtherEntities`, which is really like what the actor entity is doing (looking for other entities than itself matching something)

Reply:
`getOtherEntities` could work